libs/src/oracle-sdk-v2/services/oraclev1/gql/factory.ts

```diff
@@ -9,25 +9,13 @@ export type Config = {
   chainId: ChainId;
   address: string;
   type: OracleType;
-  addErrorMessage: (message: ErrorMessage) => void;
```

Comment:
we dont need this callback. its handled through services emitting errors in oracle context